src/translations/da_DK.json

```diff
@@ -252,7 +252,7 @@
   "ONBOARDING_DOWNLOAD_HEADLINE_PART_1": "",
   "LOGIN_APP_HEADING": "Log på",
   "LOGIN_APP_BTN_LABEL": "",
-  "LOGIN_APP_PAGE_TITLE": "",
+  "LOGIN_APP_PAGE_TITLE": "Sign in | Hedvig Insurance",
```

Comment:
This should be in danish right? Or not included

Reply:
Hm, yes I wasn't sure about that one. I _guess_ we don't need it at all since we won't use the login screen for Denmark?

Reply:
Do you think we should remove it?

Reply:
eh, let's leave it for now 🙌